src/imp/schannel.rs

```diff
@@ -305,6 +312,22 @@ impl<S: io::Read + io::Write> io::Write for TlsStream<S> {
     }
 }
 
+// SChannel-specific verification callback for validating failed certificate checks
+pub trait TlsConnectorBuilderCallbackExt {
```

Comment:
Let's just call this `TlsConnectorBuilderExt` since it may grow to include more than this method.